packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ResearchProjectForm.tsx

```diff
@@ -35,28 +35,17 @@ export type ResearchProjectFormProps = {
  */
 export default function ResearchProjectForm(props: ResearchProjectFormProps) {
   const { onSave, managedACTAccessRequirement, onHide } = props
-  const [projectLead, setProjectLead] = useState<string>('')
-  const [institution, setInstitution] = useState<string>('')
+  const [projectLead, setProjectLead] = useState<string | undefined>(undefined)
+  const [institution, setInstitution] = useState<string | undefined>(undefined)
   const [intendedDataUseStatement, setIntendedDataUseStatement] = useState<
     string | undefined
-  >(managedACTAccessRequirement.isIDURequired ? '' : undefined)
+  >(undefined)
   const [alert, setAlert] = useState<AlertProps | undefined>()
 
   const { data: existingResearchProject, isLoading: isLoadingInitialData } =
     useGetResearchProject(String(managedACTAccessRequirement.id), {
       // Infinite staleTime ensures this won't be refetched unless explicitly invalidated by the mutation
       staleTime: Infinity,
-      onSuccess: data => {
-        if (data?.projectLead) {
-          setProjectLead(data?.projectLead)
-        }
-        if (data?.institution) {
-          setInstitution(data?.institution)
-        }
-        if (data?.intendedDataUseStatement) {
-          setIntendedDataUseStatement(data?.intendedDataUseStatement)
-        }
-      },
```

Comment:
Fix the following bug:
1. Open the access request modal
2. Step to the research project form, load or fill in data
3. Close the modal
4. Open again and navigate to the same form. 

The data does not re-populate, because onSuccess is not called (this type of bug is exactly why this API is going to be removed in react query v5)


Instead use an effect that will always populate the form when appropriate.